stdlib/Random/test/OAs.jl

```diff
@@ -0,0 +1,124 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+# copied verbatim from julia/test/TestHelpers.jl
```

Comment:
Because I couldn't find the correct path to put in the `include` invocation. My short term plan was to move `OffsetArrays` from `test/TestHelpers.jl` to `stdlib/Test` as I think it's generally useful beyond `Base`, so it wouldn't be annoying to maintain two copies for a long time.

Reply:
Perhaps
```
isdefined(Main, :TestHelpers) || @eval Main include(joinpath(@__DIR__, "../../../test/TestHelpers.jl"))
using Main.TestHelpers.OAs
```

Reply:
Great, this worked! thanks

Reply:
In the end it didn't work, I had forgotten that the problem arises with on CI (with some invocation of the tests), not with `julia stdlib/Random/test/runtests.jl`. I will just do with the duplication for now.

Reply:
This worked for the REPL: https://github.com/JuliaLang/julia/pull/25544/files#diff-7b80498051b14c15cd26dd98fcc8e5acR12

Reply:
I had noticed  your solution earlier today, but I don't see how it is different from what Fredrik proposed above ?